pkg/translator/opencensus/traces_to_oc.go

```diff
@@ -24,7 +24,7 @@ import (
 	"go.opencensus.io/trace"
 	"google.golang.org/protobuf/types/known/wrapperspb"
 
-	"go.opentelemetry.io/collector/internal/occonventions"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal/occonventions"
```

Comment:
I don't see a dependency on this

Reply:
line 195 uses `val, ok := attr.Get(occonventions.AttributeSameProcessAsParentSpan)`
should it be removed from imports ?

Reply:
In the go mod

Reply:
I see. Ive added it